### PR TITLE
Adjust TPC-H testing on CI

### PR DESCRIPTION
**Pull request by hendrikmakait**
I don't see a point in testing 10 GB workloads. If we're cost-conscious, I'd rather reduce the frequency of testing.

### CODE REVIEW

File: .github/workflows/ab_tests.yml [hunk 1]
```diff
@@ -96,6 +96,12 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.RUNTIME_CI_BOT_AWS_ACCESS_KEY_ID }}
           AWS_DEFAULT_REGION: us-east-2 # this is needed for boto for some reason
           AWS_SECRET_ACCESS_KEY: ${{ secrets.RUNTIME_CI_BOT_AWS_SECRET_ACCESS_KEY }}
+          PYTHON_STUB_PAT: ${{ secrets.PYTHON_STUB_PAT }}
+          SNOWFLAKE_USER: ${{ secrets.SNOWFLAKE_USER }}
+          SNOWFLAKE_PASSWORD: ${{ secrets.SNOWFLAKE_PASSWORD }}
+          SNOWFLAKE_ACCOUNT: ${{ secrets.SNOWFLAKE_ACCOUNT }}
+          SNOWFLAKE_WAREHOUSE: ${{ secrets.SNOWFLAKE_WAREHOUSE }}
+          SNOWFLAKE_ROLE: ${{ secrets.SNOWFLAKE_ROLE }}
           AB_VERSION: ${{ matrix.runtime-version }}
```
Comment (hendrikmakait) [inline .github/workflows/ab_tests.yml:105]: I'm confused, this change should already be included in `main`: https://github.com/coiled/benchmarks/commit/2c18b816abe5cbfd84e60cacaa21c3f5030c9fa9

File: alembic/versions/2d2405ad763b_drop_tpc_h_data.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+# revision identifiers, used by Alembic.
+revision = '2d2405ad763b'
+down_revision = 'e11cd1aaed38'
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute("delete from tpch_run")
```
Comment (hendrikmakait) [inline alembic/versions/2d2405ad763b_drop_tpc_h_data.py:20]: This will drop _all_ existing records.

**Comment by fjetter** [comment]
how long is one test run (compared to the rest of the suite)?

**Comment by hendrikmakait** [comment]
Coiled estimates that the cluster cost for `test_dask.py` increase from ~$0.16 to $2.70 per run, which will add up to an increase of ~$75/month. `test_optimization.py` increases from ~$0.10 to ~$0.34, adding up to an increase of ~$7/month.

**Comment by hendrikmakait** [comment]
https://github.com/coiled/benchmarks/pull/1534 _entirely_ disables all automated tests for PRs and reduces scheduled testing to once per week. This should mitigate cost concerns.

**Review by phofl — APPROVED** [review]

**Pull request merged**